.github/workflows/build-and-push-image.yml

```diff
@@ -113,9 +113,40 @@ jobs:
               core.setFailed(error.message);
             }
 
+  acr-import:
+    name: Import to images to ${{ needs.set-env.outputs.environment }} ACR
+    needs: [ build-and-push-image, set-env ]
+    runs-on: ubuntu-22.04
+    if: needs.set-env.outputs.environment == 'development'
+    environment: ${{ needs.set-env.outputs.environment }}
+      - name: Azure login with ACA credentials
+        uses: azure/login@v2
+        with:
+          creds: ${{ secrets.ACA_CREDENTIALS }}
+
+      - name: Run ACR Import
+        uses: azure/cli@v2
+        with:
+          azcliversion: 2.40.0
+          inlineScript: |
+            SOURCES=(
+              ghcr.io/${{ needs.set-env.outputs.github_repository_lc }}:${{ needs.set-env.outputs.branch }}
+              ghcr.io/${{ needs.set-env.outputs.github_repository_lc }}:${{ needs.set-env.outputs.release }}
+              ghcr.io/${{ needs.set-env.outputs.github_repository_lc }}:latest
+            )
+            az config set extension.use_dynamic_install=yes_without_prompt
+            for source in "${SOURCES[@]}" do
+            az acr import \
+              --name ${{ secrets.ACR_NAME } \
+              --source "$source" \
+              --image sourceimage:tag \
```

Comment:
`sourceimage:tag` seems like something you forgot to update?

Reply:
Ah yep ...

Reply:
I've had to change `SOURCES` to `TAGS`, so that the for loop works correctly, and uses the same tag for both `source` and `image`